### ISSUE TITLE
Build failure (missing dependency wingpanel-2.0)

### ISSUE THREAD

**Issue by waldyrious** [PlugaruT/wingpanel-indicator-sys-monitor#42]
I tried installing this on a Juno system but got an error when running `meson build`:

```
$ git clone https://github.com/PlugaruT/wingpanel-indicator-sys-monitor.git
$ cd wingpanel-indicator-sys-monitor;
$ sudo apt install libgtop2-dev libgranite-dev libgtk-3-dev libwingpanel-2.0-dev meson valac
$ meson build --prefix=/usr
```

The output of the `meson build` command was:

```
The Meson build system
Version: 0.49.2
Source dir: /home/waldyrious/repos/wingpanel-indicator-sys-monitor
Build dir: /home/waldyrious/repos/wingpanel-indicator-sys-monitor/build
Build type: native build
Project name: com.github.plugarut.wingpanel-indicator-sys-monitor
Project version: undefined
Native C compiler: cc (gcc 7.4.0 "cc (Ubuntu 7.4.0-1ubuntu1~18.04.1) 7.4.0")
Native Vala compiler: valac (valac 0.40.18)
Build machine cpu family: x86_64
Build machine cpu: x86_64
Found pkg-config: /home/waldyrious/.linuxbrew/bin/pkg-config (0.29.2)
Found CMake: /home/waldyrious/.linuxbrew/bin/cmake (3.15.5)
Dependency wingpanel-2.0 found: NO (tried pkgconfig and cmake)

meson.build:10:0: ERROR:  Dependency "wingpanel-2.0" not found, tried pkgconfig and cmake

A full log can be found at /home/waldyrious/repos/wingpanel-indicator-sys-monitor/build/meson-logs/meson-log.txt
```

Maybe the [list of dependencies in the README](https://github.com/PlugaruT/wingpanel-indicator-sys-monitor#building-and-installation) is not complete?

**Comment by PlugaruT** [comment]
Hi and thanks for reporting this issue.
I will try to replicate this on my machine and come back with a solution.

**Comment by waldyrious** [comment]
Update: I ended up bumping into a similar issue for another repo. According to [this comment](https://github.com/phase1geo/Outliner/issues/5#issuecomment-604070838) from the thread where I reported it:

> Homebrew's `pkg-config` only reports packages installed by Homebrew

For me the solution was to use the distro's `pkg-config` instead of Homebrew's. I ran `brew uninstall pkg-config` (I didn't need it anymore, but I guess I could have simply done `brew unlink pkg-config` if I wanted to just remove it from the path temporarily).

I can now build the project!
